src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -220,13 +242,52 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                         <div>          <span>REPO_TOKEN: {"${{ secrets.GITHUB_TOKEN }}"}</span></div>
                       </Tooltip>
                     </Code>
-
+        
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative/cml_dvc_case/pull/4">
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-dvc/pull/1">
                         <Image src="/img/github/dvc-report.png" alt="GitHub DVC report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-dvc">
```

Comment:
- [ ] TODO: create this repo properly @0x2b3bfa0

```suggestion
                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-ai/cml-example-dvc">
```

Reply:
```suggestion
                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-ai/cml-dvc-case">
```